ui/packages/app/src/project_setting/form/ProjectForm.js

```diff
@@ -46,6 +46,12 @@ const ProjectForm = () => {
   };
 
   const [isValidStream, setIsValidStream] = useState(false);
+
+  useEffect(() => {
```

Comment:
why is this needed while for labels and stream the `useEffect()` is not? Generally we should avoid `useEffect` unless necessary.

If the child component needs to know the validity, would something like this work? 

```
const isValidTeam = team => ...
 isValidTeam={isValidTeam(project.name)}
```

Reply:
Hmm if I understand correctly, I think it's just to check if the team's value is empty or not. In that case I think there's already this effect within the `Team` component: https://github.com/caraml-dev/mlp/blob/main/ui/packages/app/src/project_setting/form/Team.js#L34 so I wonder if we need this additional check here 🤔

Reply:
yep @deadlycoconuts is right, this effect is already in `Team.js`, have removed it